src/DotVVM.Framework/Controls/UpdateProgress.cs

```diff
@@ -13,6 +14,20 @@ namespace DotVVM.Framework.Controls
     /// </summary>
     public class UpdateProgress : HtmlGenericControl
     {
+
+        /// <summary>
+        /// Gets or sets the delay after which the content inside UpdateProgress control is shown
```

Comment:
There should be hinted in which a time unit (s, **ms**, ns) is the value expected